src/solvers/solveCobraLPCPLEX.m

```diff
@@ -179,6 +179,9 @@
 %conflict refinement is desired in the case that infeasibility is detected
 %by CPLEX.
 if conflictResolve
+    % set to deterministic mode to get reproducible conflict resolve file
+    cpxControl.PARALLEL = 1; % Tomlab CPLEX interface

```

Comment:
This is actually inverted:
```
cpxControl.PARALLEL = 1; % ILOG CPLEX interface
cpxControl.PARALLELMODE = 1; % Tomlab CPLEX interface
```
[Reference ILOG](http://www.ibm.com/support/knowledgecenter/en/SS9UKU_12.4.0/com.ibm.cplex.zos.help/Parameters/topics/ParallelMode.html) and [Reference Tomlab](http://tomwiki.com/CPLEX_Parallel_Mode)

Can you add a test?